tests/utils/factories.py

```diff
@@ -19,7 +21,32 @@ def symbol_graph_static_test() -> SymbolGraph:
     # assuming the path to a valid index protobuf file, you should replace it with your own file path
     file_dir = os.path.dirname(os.path.abspath(__file__))
     index_path = os.path.join(file_dir, "..", "test.scip")
-    return SymbolGraph(index_path)
+    return SymbolGraph(index_path, pickle_graph=False)
+
+
+@pytest.fixture
+def static_symbol_graph_static_test() -> SymbolGraph:
```

Comment:
@emrgnt-cmplxty The naming of `symbol_graph_static_test` for the dynamic tests and now `static_symbol_graph_static_test` seems redundant. Thoughts on making these `symbol_graph_dynamic_test` and `symbol_graph_static_test`?

Reply:
hmm, one of these 'static' graphs is actually static because there are indices in the test suite. Maybe let's call it static_indices_graph?